addons/l10n_br/models/res_company.py

```diff
@@ -12,7 +12,12 @@ class ResCompany(models.Model):
     l10n_br_ie_code = fields.Char(string="IE", help="State Tax Identification Number. Should contain 9-14 digits.") # each state has its own format. Not all of the validation rules can be easily found.
     l10n_br_im_code = fields.Char(string="IM", help="Municipal Tax Identification Number") # each municipality has its own format. There is no information about validation anywhere.
     l10n_br_nire_code = fields.Char(string="NIRE", help="State Commercial Identification Number. Should contain 11 digits.")
+    show_taxes = fields.Boolean(compute='_compute_show_taxes', store=False)
```

Comment:
This doesn't work with inheritance.
If you install BR and US, what will the behavior be? :wink:

Reply:
Yes, it only works for the last installed either US or BR. Because of the computed method change.
Is that right?

Reply:
This is actually a behavior that I didn't expect from the computed fields, but Am I now forced to have this column separated into one per localization pack e.g., l10n_us & l10n_br? 
Like I'm trying to escape that by keeping it generic like having a field that every localization pack can set and every report/template decide to show the column or not based on it.

Reply:
Indeed.
So for it to work, you would need to make the method inheritable from one of the common modules, in the case `account`

Reply:
Do you mean like having this computed method where I'm defining `show_taxes` (res.company in account module) and extend it in the localization package? or you mean another thing. Sorry but I think I'm missing something. If you can elaborate more what you mean by making the method inheritable that will be awesome!

Reply:
Just defined a `_compute_show_taxes` method in `account` and call `super()._compute_show_taxes` in the l10n

Reply:
Yes, This is what I tried to do but it seems producing the same issue, that's why i thought i missed something. I defined the method in account module where show_taxes column exists and in the l10n just called the super method.

Reply:
Done!
I had to make it stored column tho.